servicecontrol/audit-instances/creating-config-file.md

```diff
@@ -245,9 +245,24 @@ Type: string
 
 ## Troubleshooting
 
-#### ServiceControl.Audit/ExposeRavenDB
 
-ServiceControl Audit stores its data in a RavenDB embedded instance. By default, the RavenDB instance is accessible only by the ServiceControl Audit service. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running, ServiceControl Audit can be configured to expose the RavenDB studio.
+ServiceControl Audit stores its data in a RavenDB embedded instance. If, during troubleshooting, direct access to the RavenDB instance is required while ServiceControl Audit is running use the following instructions.
+
+### Version 4.26 and above
```

Comment:
Do we want to mention here, ner persistance? As you can have the old instance upgraded to the 4.26

Reply:
Good catch, I have reworked the sections to convey this now